app/pandas.ipynb

```diff
@@ -424,9 +541,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# filter s1 here"
-   ],
+   "source": "even_s1 = s1[s1 % 2 == 0]",
```

Comment:
The filtering of even numbers from s1 is defined but not executed. You should print the result to verify the operation.